pkg/routing/algorithm_prophet.go

```diff
@@ -1,19 +1,17 @@
 // SPDX-FileCopyrightText: 2019 Markus Sommer
+// SPDX-FileCopyrightText: 2019, 2021 Markus Sommer
```

Comment:
Nit: Just add the year to the line above?

Reply:
oops, yeah, don't know how that happened....

Reply:
@mogren: Thank you very much for having a look at it.